routers/repo/release.go

```diff
@@ -167,6 +168,17 @@ func NewReleasePost(ctx *context.Context, form auth.NewReleaseForm) {
 		return
 	}
 
+	tagCreationDate := time.Now()
```

Comment:
You can store a `int64` type variable here by `var tagCreatedUnix int64`, so it defaults to `0`.